### PR TITLE
Feature/webconf

### PR DESCRIPTION
**Pull request by nilcsi**

### CODE REVIEW

File: airmet/web_conf.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+site += "<p>Voltage: (String)voltage V</p>\n";
+site += "<form action= method=post>\n";
+site += "\t<label>Temperature offset:</label>\n";
+site += "<input maxlength=30 name=\"tempcalib\" value=\"(String)temperature\"><br>\n";
+site += "<label>Pressure offset:</label>\n";
+site += "<input maxlength=30 name=\"prescalib\" value=\"(String)pressureOffset\"><br>\n";
+site += "\t<label>Voltage multiplier:</label>\n";
+site += "<input maxlength=30 name=\"voltagemulti\" value=\"(String)voltage\"><br>\n";
+site += "\t<label>Thingspeak API key:</label>\n";
+site += "<input maxlength=30 name=\"thinspeak\" value=\"bigybula\"><br>\n";
```
Comment (dezur) [inline airmet/web_conf.h]: typo: tingspeak

File: airmet/web_conf.h [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+site += "<title>ESP8266 Web Form Demo</title>\n";
+site += "<style>\n";
+site += "\t\tlabel{\n";
+site += "\t\t\tline-height: 30px;\n";
+site += "\t\t\t}\n";
+site += "</style>\n";
+site += "</head>\n";
+site += "<body>\n";
+site += "<h1>AIRMET</h1>\n";
+site += "<p>Temperature: (String)temperature °C</p>\n";
```
Comment (dezur) [inline airmet/web_conf.h]: Remove degree symbol

Comment (nilcsi) [reply airmet/web_conf.h]: done

File: airmet/web_conf.h (hunk 1 above)
Comment (nilcsi) [reply airmet/web_conf.h]: done

**Pull request merged**